VimeoUpload/Descriptor System/KeyedArchiver.swift

```diff
@@ -65,3 +68,23 @@ public class KeyedArchiver: ArchiverProtocol
         return URL.absoluteString! as String
     }
 }
+
+// MARK: NSKeyedArchiver Migration
+
+private extension KeyedArchiver
```

Comment:
@nicolelehrer Ah you're right I forgot about the cache. There's probably a better place to put this in that case.